2_TLDR_Short_List.md

```diff
@@ -55,8 +58,8 @@ It lays out the 20 most essential security + privacy tips, that you should compl
 Switch to alternative open-source, privacy-respecting apps and services, which won't collect your data, track you or show targetted ads
 
 #### Security
-- Password Managers: [BitWarden] | [1Password] | [KeePassXC] *(offline, no-sync)*
-- 2-Factor Authentication: [Aegis] *(Android)* | [Authenticator] *(iOS)* | [AndOTP] *(Android)*
+- Password Managers: [BitWarden] | [Pass] | [KeePassXC] *(offline, no-sync)*
```

Comment:
I think we're missing a link here in preview. How do I fix it? It has no clear link in source but there's one in markdown view.